dcm4che-core/src/main/java/org/dcm4che3/io/DicomInputStream.java

```diff
@@ -709,10 +795,16 @@ private void checkIsThis(DicomInputStream dis) {
             throw new IllegalArgumentException("dis != this");
     }
 
-    private void skipAttribute(String message) throws IOException {
-        LOG.warn(message,
-                 new Object[] { TagUtils.toString(tag), length, tagPos });
-        skip(length);
+    /**
+     * @param message the message to use in the warning log message
+     * @param methodName the name of the method that is skipping the attribute
+     * @throws IOException potentially thrown when performing the 'skip' operation
+     */
+    private void skipAttribute(String message, String methodName) throws IOException {
+        String tagAsString = TagUtils.toString(this.tag);
+        LOG.warn(message, tagAsString, length, tagPos, methodName);
+        long skipLength = skip(length);
```

Comment:
Hmm, I wonder if this call to skip(.) here shouldn't actually be a call to skipFully(.)? (@gunterze)

Reply:
I'm going to resolve this conversation so that I can merge my pull request. I don't like this feature of github, you should be able to merge even if there are comments in the pull request. Assuming the reviewer has already approved it.